src/Codeception/TestRunner.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Codeception;
+
+use Codeception\PHPUnit\FilterTest;
+use PHPUnit\Framework\TestResult;
+use PHPUnit\Runner\Filter\ExcludeGroupFilterIterator;
+use PHPUnit\Runner\Filter\Factory;
+use PHPUnit\Runner\Filter\IncludeGroupFilterIterator;
+use ReflectionProperty;
+
+class TestRunner
+{
+    protected array $config = [];
+
+    public function __construct()
+    {
+        $this->config = Configuration::config();
+    }
+
+    public function prepareSuite(\PHPUnit\Framework\Test $suite, array $arguments)
```

Comment:
This test contains some remnants of src/PHPUnit/Runner.php, but it isn't a Test Runner.

All this code could be moved to SuiteManager which is a sole consumer.

The real TestRunner could be extracted from `run` and `runPhpUnitTest` methods in Codeception\Suite.

Reply:
I eliminated this class.